src/ast/enrichment/annotate_projected_named_paths.c

```diff
@@ -7,6 +7,8 @@
 #include "annotate_projected_named_paths.h"
 #include "../../util/arr.h"
 
+static void _annotate_projected_named_path(AST *ast);
```

Comment:
remove this line, move implementation upwards.

Reply:
`_annotate_projected_named_path()` and `_annotate_foreach_clause_projected_named_path()` call each other, so one of their declarations must be forwarded.